### DISCUSSION TITLE
Upload/FTP Delta Script

### DISCUSSION THREAD

**Discussion by w951wcl** [thumbsup/thumbsup#247] [Q&A]
Is there a template script to FTP the changes to an Gallery after adding a few new folders?
1) Media Large sorted by data modified
2) Media small sorted by data modified
3) Media thumbs 
4) then all the HTML?

Just wondering what everyone is doing.

Thanks

**Comment by rprieto** [comment]
Hi, that's an interesting discussion to have.

I personally upload in 2 passes: everything except HTML, and then the albums. This makes sure people don't see blank thumbnails if they browse in the middle of an update.

I don't use FTP, but I recommend letting a sync tool figure out what to sync. For example you can use https://rclone.org/commands/rclone_sync/.

**Comment by cyruz333** [comment]
>
